.aegir.js

```diff
@@ -0,0 +1,15 @@
+// This config is fixing deepmerge's issue https://github.com/KyleAMathews/deepmerge/issues/87.
+// We can remove it after fix webpack's issue https://github.com/webpack/webpack/issues/6584.
+
+const path = require('path')
+const config = {
+  webpack: {
+    resolve: {
+      alias: {
+        deepmerge$: path.resolve(__dirname, 'node_modules/deepmerge/dist/umd.js')
```

Comment:
Do we really need `deepmerge`, can we replace it with lodash merge or defaultsDeep?

Reply:
Thanks for comment. Maybe we can replace `deepmerge` (Or, `Object.assign()` ?) . I'll investigate that.

Reply:
`Object.assign` does NOT do a deep merge.

Reply:
LGTM